docs/source/bundle.rst

```diff
@@ -32,3 +32,21 @@ Model Bundle
 ---------------
 .. autoclass:: ConfigParser
     :members:
+
+`Scripts`
+---------
+.. autofunction:: run
+
+`Utilities`
+-----------
+.. autofunction:: load_config_file
+
+.. autofunction:: load_config_file_content
```

Comment:
Form a use's perspective I don't know the difference of load_config_file and load_config_file_content, parse_config_file. Could you revise all the function names in utilities if they are frequently used?

Reply:
Hi @wyli ,

Thanks for your suggestion, that's a good point.
I totally refactored the config reading logic in the latest commit.

Thanks.